docs/Webhooks/Geo-triggering.md

```diff
@@ -1,183 +1,207 @@
+2. Select the desired version (`1.0` or `2.0`) for the Geo-triggering event.
+3. Save your changes.
+
+![](../assets/Webhook%20-%20geotriggering%202.png)
+
+## Geo-triggering event (V2.0)
 
-From the Canvas Integration section, add a new Webhook and select Geo-trigger as the type in the dropdown.
+The `2.0` version of the webhooks payload for Geo-triggering events includes a standardized structure that aligns with the Analytics API and SDK Entry and Exit events. This version provides additional data points for enhanced insights and uniformity.
 
-![](../assets/Add-New-Geo-trigger-webhook-1024x750.png)
+### Root Level Fields
 
-### Additional Header and Body fields
+This table lists the primary fields found at the root level of the payload.
 
-You can have additional information in either the headers or the body of the webhook’s request by adding extra fields.
+| Field          | Description                                                   |
+|----------------|---------------------------------------------------------------|
+| `ulid`         | Unique identifier for the event.                              |
+| `retryCount`   | The number of retry attempts for the webhook delivery.        |
+| `triggerChainId` | Identifier for the chain of triggers leading to this event. |
+| `notificationType` | Type of notification (e.g., exit).                       |
+| `receivedAt`   | Timestamp when the event was received.                        |
+| `submissionTime` | Timestamp when the event was submitted.                    |
+| `accountId`    | Account identifier.                                           |
+| `installRef`   | Reference ID for the installation.                            |
+| `projectId`    | Project identifier.                                           |
 
-![](../assets/Add-extra-headers-and-body-fields-1024x647.png)
 
-You can also configure a Geo-trigger Webhook through Config API (Information on using the Config API for registering a Webhook can be found [here](https://config-docs.bluedot.io/#operation/addProject).)
+### `appInfo` fields
 
-If you’d like to Create a Destination using [Config API](../APIs/Config%20API/Overview.md), you’ll use the following endpoints:
+This table details the fields within the `appInfo` object, providing information about the application.
 
+| Field                                     | Description                                                                 |
+|-------------------------------------------|-----------------------------------------------------------------------------|
+| `appInfo.appBuildVersion`                 | Build version of the application.                                           |
+| `appInfo.customEventMetaData`             | Custom metadata related to the event.                                        |
+| `appInfo.customEventMetaData.[KEY]` | The key of the custom event data key/value pair.                                               |
```

Comment:
Description should be the **value** of the custom event data key/value pair - all of the other descriptions describe the value.

Reply:
Updated!